src/store/src/Document/Vectorizer.php

```diff
@@ -14,16 +14,17 @@
 use Psr\Log\LoggerInterface;
 use Psr\Log\NullLogger;
 use Symfony\AI\Platform\Capability;
+use Symfony\AI\Platform\Exception\ExceptionInterface;
 use Symfony\AI\Platform\PlatformInterface;
 use Symfony\AI\Platform\Vector\Vector;
 use Symfony\AI\Store\Exception\RuntimeException;
 
-final readonly class Vectorizer implements VectorizerInterface
+final class Vectorizer implements VectorizerInterface
 {
     public function __construct(
-        private PlatformInterface $platform,
-        private string $model,
-        private LoggerInterface $logger = new NullLogger(),
+        private readonly PlatformInterface $platform,
+        private readonly string $model,
+        private readonly LoggerInterface $logger = new NullLogger(),
     ) {
     }
```

Comment:
a bit confused about this diff, since it is already fixed on main with #829 

https://github.com/symfony/ai/blob/5f498f5d46af215f9a4eb3b26cefb8a4d67d5fb1/src/store/src/Document/Vectorizer.php#L21-L28

Reply:
Yeah weird, I'll try rebase although that shouldn't have any bearing on this diff... Maybe GH being buggy.